buildSrc/src/main/groovy/org.apache.groovy-tested.gradle

```diff
@@ -32,7 +32,7 @@ sourceSets {
     }
 }
 
-tasks.withType(Test).configureEach {
+def configureTest = { Test it ->
     def fs = objects.newInstance(TestServices).fileSystemOperations
     def grapeDirectory = new File(temporaryDir, "grape")
     def jdk8 = ['-XX:+UseConcMarkSweepGC']
```

Comment:
* What went wrong:
Could not determine the dependencies of task ':testMatrix'.
> Could not create task ':testJdk8'.
   > Could not get unknown property 'temporaryDir' for root project 'groovy' of type org.gradle.api.Project.